src/Sarif/Schemata/sarif-external-property-file-2.1.0-rtm.5.json

```diff
@@ -17,13 +17,13 @@
     },
 
     "guid": {
-      "description": "A stable, unique identifer for this external properties object, in the form of a GUID.",
+      "description": "A stable, unique identifier for this external properties object, in the form of a GUID.",
```

Comment:
>identifier [](http://example.com/codeflow?start=39&length=10)

Oops, we probably should create an rtm6 version of this file which only has this spelling change added.